### PR TITLE
Pytorchlightin

### LINKED ISSUE #7: PyTorch Lightning models for MNIST and galaxy zoo.

### LINKED ISSUE #9: Real time metric plot.

### PR DESCRIPTION
**Pull request by pierpaolo28**
In this branch, the following additions have been included:
- Dataset loading procedure
- Training loop using PyTorch Lighting
- Usage example notebook 
- Live tracker training/validation loss
- Updated requirements.txt to include additional used libraries

### CODE REVIEW

**Comment by yannidd** [comment]
We'll open a new PR for ONNX (or whatever else we end up using) to merge this branch when it's ready since it's no longer about pytorch-lightning integration. Closing this now.
